src/onboarding/ha-onboarding.ts

```diff
@@ -256,9 +283,16 @@ class HaOnboarding extends litLocalizeLiteMixin(HassElement) {
       step.step === stepResult.type ? { ...step, done: true } : step
     );
 
-    if (stepResult.type === "user") {
+    if (stepResult.type === "init") {
+      this._init = false;
+      this._restoring = stepResult.result.restore;
+      if (!this._restoring) {
+        this._progress = 0.25;
```

Comment:
Can we compute progress here too?

Reply:
Why? We know exactly where we are, especially when we will add back navigation, the step calculation will not work anymore